### PR TITLE
Document how to run unit tests

### PR DESCRIPTION
**Pull request by JimStockwell**
Explicitly show how to run unit tests and checkstyle.

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -96,7 +96,13 @@ The project can be built from the root directory using the standard Gradle comma
 	$ ./gradlew build
 ----
 
+=== Running Unit Tests
+Unit tests and checkstyle can be run from the relevant *module directory* using the standard Gradle command:
 
+[indent=0]
+----
+        $ ../../gradlew check test
```
Comment (wilkinsona) [inline CONTRIBUTING.adoc:104]: `check` depends on `test` so running both isn't necessary. `check` alone is sufficient.

I'm also not sure about running from a particular module directory. The style that I see more frequently is to run from the root using a task's full path, for example:

```
./gradlew spring-boot-project:spring-boot:check
```

**Review by wilkinsona — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (wilkinsona) [reply CONTRIBUTING.adoc:104]: Looking at https://github.com/spring-projects/spring-boot/issues/24299 reminded me that running both `check` and `test` is necessary when using `--tests`. IIRC, there's a Gradle issue about this but I can't find it right now. That's quite specialised usage so, IMO, we should be sticking to the single `check` task here.

Comment (JimStockwell) [reply CONTRIBUTING.adoc]: That makes a lot of sense to me @wilkinsona -- thank you.  I've incorporated those changes.  In light of the insight you brought, I made some other small changes to this pull request, which you will see.  If there's any concern with them, please let me know.  Thanks again.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -66,6 +66,8 @@ added after the original pull request but before a merge.
   than cosmetic changes).
 * Add some Javadocs.
 * A few unit tests would help a lot as well -- someone has to do it.
+* Verification tasks, including tests and stylecheck, can be executed
```
Comment (wilkinsona) [inline CONTRIBUTING.adoc:69]: Typo? Should be Checkstyle rather than stylecheck, I think.

Comment (JimStockwell) [reply CONTRIBUTING.adoc:69]: Ouch!  Thanks! :)

**Comment by philwebb** [comment]
Thanks very much for PR. I've merged this to master.

**Pull request closed without merging**